apps/projects/contracts/Projects.sol

```diff
@@ -549,48 +548,47 @@ contract Projects is IsContract, AragonApp {
      * @return  applicant address
      */
     function getApplicant(
-        bytes32 _repoId, 
-        uint256 _issueNumber, 
+        bytes32 _repoId,
+        uint256 _issueNumber,
         uint256 _idx
-    ) public view returns(address applicant,string application) 
+    ) public view returns(address applicant,string application)
     {
         applicant = repos[_repoId].issues[_issueNumber].applicants[_idx];
         application = repos[_repoId].issues[_issueNumber].assignmentRequests[applicant];
     }
 
-    ///**
-    // * @notice Returns Applicant's Github Username
-    // * @param _repoId the github repo id of the issue
-    // * @param _issueNumber the github issue up for assignment
-    // * @param _applicant the address of the applicant
-    // * @return  application IPFS hash for the applicant's proposed timeline and strategy
-    // */
-    //function getAssignmentRequest(
-    //    bytes32 _repoId, 
-    //    uint256 _issueNumber, 
-    //    address _applicant
-    //) public view returns(string application) 
-    //{
-    //    application = repos[_repoId].issues[_issueNumber].assignmentRequests[_applicant];
-    //}
+        /**
+     * @notice Returns Applicant array length
+     * @param _repoId the github repo id of the issue
+     * @param _issueNumber the github issue up for assignmen
+     * @return  array length of the applicants array
+     */
+    function getSubmissionsLength(
+        bytes32 _repoId,
+        uint256 _issueNumber
+    ) public view returns(uint256 applicantQty)
+    {
+        applicantQty = repos[_repoId].issues[_issueNumber].submissionIndices.length;
+    }
 
     /**
      * @notice Returns contributor's work submission
      * @param _repoId the github repo id of the issue
      * @param _issueNumber the github issue being worked on
-     * @param _contributor the address of the contributor
+     * @param _submissionNumber the address of the contributor
```

Comment:
We should update the text for the param type (submission number instead of address)